### PR TITLE
Fixed robot commander to accept partial traj and added examples

### PR DESCRIPTION
**Pull request by beatrizleon**
Robot commander sends now partial trajectories and velocities are not setted to zero unless specified.
Added an example to do a grasp and in the middle the index is stoping with partial traj.
Added an example to show the way partial trajectories work showing graphs of the trajectories generated. Only on joint of the index and middle finger are moved. Here is a video to show the example:
gifrecord_2015-08-11_134851

### CODE REVIEW

File: sr_example/scripts/sr_example/commander/sr_right_hand_partial_traj_grasp.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    trajectory_point.time_from_start = rospy.Duration.from_sec(float(duration))
+    trajectory_point.positions = []
+    trajectory_point.velocities = []
+    trajectory_point.accelerations = []
+    trajectory_point.effort = []
+    for key in joint_trajectory.joint_names:
+        trajectory_point.positions.append(posture[key])
+        trajectory_point.velocities.append(0.0)
+        trajectory_point.accelerations.append(0.0)
+        trajectory_point.effort.append(0.0)
```
Comment (toliver) [inline sr_example/scripts/sr_example/commander/sr_right_hand_partial_traj_grasp.py]: Maybe we shouldn't add vel, acc and effort targets in this example?

File: sr_example/scripts/sr_example/commander/sr_right_hand_partial_traj_grasp.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+from sr_robot_commander.sr_hand_commander import SrHandCommander
+from trajectory_msgs.msg import JointTrajectory, JointTrajectoryPoint
+
+rospy.init_node("set_traj_goals", anonymous=True) 
+rospy.sleep(1)  # Do not start with zero
+
+def construct_trajectory_point(posture, duration):
+    trajectory_point = JointTrajectoryPoint()
+    trajectory_point.time_from_start = rospy.Duration.from_sec(float(duration))
+    trajectory_point.positions = []
```
Comment (toliver) [inline sr_example/scripts/sr_example/commander/sr_right_hand_partial_traj_grasp.py]: This shouldn't be necessary, as the constructor of JointTrajectoryPoint does it already.

**Comment by beatrizleon** [comment]
Fixed comments

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py [hunk 3]
```diff
@@ -268,10 +226,23 @@ def move_to_joint_value_target_unsafe(self, joint_states, time=0.002, wait=True)
...
+        #self._update_default_trajectory()
+        #self._set_targets_to_default_trajectory(joint_states)
+        goal = FollowJointTrajectoryGoal()
+        goal.trajectory.joint_names = list(joint_states.keys())
+        point = JointTrajectoryPoint()
+        point.time_from_start = rospy.Duration.from_sec(time)
+        point.positions = []
+        point.velocities = []
+        point.accelerations = []
+        point.effort = []
```
Comment (toliver) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: Same here. we don't need to initialize.

**Comment by toliver** [comment]
Also please check roslint compliance on the new files.

**Comment by beatrizleon** [comment]
Done

**Comment by beatrizleon** [comment]
Fixed #358

**Pull request merged**